templates/bbb-web.properties

```diff
@@ -11,9 +11,10 @@ muteOnStart={{ bbb_mute_on_start | ternary("false", "true") }}
 appLogLevel={{ bbb_app_log_level }}
 maxInactivityTimeoutMinutes={{ bbb_meeting_inactivity_timeout_minutes }}
 defaultDialAccessNumber={{ bbb_sip_pretty_number }}
-{% if bbb_sip_pretty_number %}
-# TODO make defaultWelcomeMessageFooter customizable via variable in defaults - this way its static.
-defaultWelcomeMessageFooter=Um dieser Konferenz per Telefon beizutreten, w&auml;hlen Sie <tt>%%DIALNUM%%</tt> und geben <tt>%%CONFNUM%%</tt> als Konferenz-PIN ein.<br><br><b>Hinweis:</b> Anrufer sind ggf. stummgeschaltet. Dr&uuml;cken Sie <tt>0</tt> um zu sprechen bzw. sich stummzuschalten.'
+{% if bbb_dialin_enabled and bbb_dialin_overwrite_footer %}
+defaultWelcomeMessageFooter={{ bbb_dialin_footer }}
+{% else %}
+defaultWelcomeMessageFooter={{ bbb_default_welcome_message_footer }}
```

Comment:
I am not fluent in jinja templateing, if someone could check this? There might be a better way to do this.
Might be even completely wrong 🤷🏻‍♂️

Reply:
you could write a more complicated one-liner, but I think this this is a good solution